src/asynk/async_worker.rs

```diff
@@ -1,79 +1,73 @@
-use crate::asynk::async_queue::AsyncQueue;
 use crate::asynk::async_queue::AsyncQueueable;
 use crate::asynk::async_queue::FangTaskState;
 use crate::asynk::async_queue::Task;
+use crate::asynk::async_queue::DEFAULT_TASK_TYPE;
 use crate::asynk::async_runnable::AsyncRunnable;
 use crate::asynk::Error;
 use crate::{RetentionMode, SleepParams};
-use bb8_postgres::tokio_postgres::tls::MakeTlsConnect;
-use bb8_postgres::tokio_postgres::tls::TlsConnect;
-use bb8_postgres::tokio_postgres::Socket;
 use log::error;
 use std::time::Duration;
 use typed_builder::TypedBuilder;
 
-#[derive(TypedBuilder, Debug)]
-pub struct AsyncWorker<Tls>
-where
-    Tls: MakeTlsConnect<Socket> + Clone + Send + Sync + 'static,
-    <Tls as MakeTlsConnect<Socket>>::Stream: Send + Sync,
-    <Tls as MakeTlsConnect<Socket>>::TlsConnect: Send,
-    <<Tls as MakeTlsConnect<Socket>>::TlsConnect as TlsConnect<Socket>>::Future: Send,
-{
-    #[builder(setter(into))]
-    pub queue: AsyncQueue<Tls>,
+#[derive(TypedBuilder)]
+pub struct AsyncWorker<'a> {
     #[builder(setter(into))]
+    pub queue: &'a mut dyn AsyncQueueable,
+    #[builder(default=Some(DEFAULT_TASK_TYPE.to_string()) , setter(into))]
```

Comment:
if it always has a default value, maybe it doesn't need `Option`?

Reply:
deleting in [1945344](https://github.com/ayrat555/fang/pull/46/commits/194534459c67e042eff605b1b1798a260ca277f8)